packages/client/terarium/package.json

```diff
@@ -4,24 +4,24 @@
 	"version": "0.0.0",
 	"type": "module",
 	"scripts": {
-		"dev": "vite",
+		"dev": "PORT=8080 vite",
```

Comment:
extract into an .env file for use in both server and client

Reply:
I don't think we need a .env file for this, but I did move specifying the port to the vite config and I set strictPort to true so that vite ensures we run on 8080 and will exit if it can't run on 8080.

Reply:
details... still feel that it may be good to have a local config that is changeable that can affect both as they need to be in sync... but for now fine... still need to settle on the project setup for all this then see what makes the most sense

Reply:
Yes, I agree, once this is merge and we create a **TERArium** repo that includes scripts to install and run the whole project at different level, the port numbers should be define then